XenAdmin/Wizards/PatchingWizard/AutomatedUpdatesBasePage.cs

```diff
@@ -434,7 +439,7 @@ private void WorkerCompleted(object sender, RunWorkerCompletedEventArgs e)
         {
             if (e.Cancelled)
             {
-                Status = Status.Completed;
+                Status = Status.Cancelled;

```

Comment:
This change is not related to the current bug, however I think it was a mistake that we were setting it to Completed.